src/Containers/HorizontalScrollList.tsx

```diff
@@ -0,0 +1,121 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { flex, scroll } from '../Utils/Mixins';
+import { Select } from '../Inputs/Select';
+
+export interface ScrollListProps extends React.HTMLAttributes<HTMLDivElement> {
+    labelArray: string[];
+    menuName: string;
+    menuWidth: number;
+    labelPadding: string;
+    hoveredStyle?: Function;
+    selectedStyle?: Function;
+    displaySelected?: boolean;
+    displayDropDown?: boolean;
+}
+
+export interface ScrollListDivProps {
+    menuWidth?: number;
+}
+
+export interface ListItemProps {
+    label: string;
+    hoveredStyle: Function;
+    selectedStyle: Function;
+    isSelected: boolean;
+}
+
+export const HorizontalScrollList: React.FC<ScrollListProps> = ({
+    labelArray,
+    menuName = 'Menu',
+    hoveredStyle,
+    menuWidth,
+    selectedStyle,
+    displaySelected = true,
+    displayDropDown = true,
+    ...props
+}): React.ReactElement => {
+    const [selectedLabel, setSelectedLabel] = useState('');
+    return (
+        <HorizontalListDiv {...props}>
+            {displayDropDown && (
+                <DropDownDiv menuWidth={menuWidth}>
+                    <Select
+                        placeholder={menuName}
+                        onChange={({ target }: { target: any }) => {
+                            setSelectedLabel(target.value);
+                        }}
+                        value={displaySelected ? selectedLabel : menuName}
+                    >
+                        {labelArray.map(label => (
+                            <option value={label}>{label}</option>
+                        ))}
+                    </Select>
+                </DropDownDiv>
+            )}
+            <HorizontalList>
+                {labelArray.map((label, index) => (
+                    <HorizontalListItem
+                        key={label + index}
+                        onClick={() => setSelectedLabel(label)}
+                        label={label}
+                        hoveredStyle={
+                            hoveredStyle ? hoveredStyle : defaultHoveredStyle
```

Comment:
this is a CSS thing not a JS thing, put an isHovering Prop and Style it in the CSS styles component.

Reply:
Or just use isHover from CSS?